docusaurus.config.js

```diff
@@ -7,7 +7,7 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 /** @type {import('@docusaurus/types').Config} */
 const config = {
   title: 'wasmCloud',
-  tagline: 'Develop. Deploy. Distribute.',
+  tagline: 'Build applications in any language and deploy them anywhere.',
```

Comment:
What do you think about "build distributed applications..."?

Reply:
I do like that

Reply:
hrm, somewhat redundant with "deploy them anywhere"

Reply:
@ricochet I disagree. One technology could make it easy to deploy monoliths anywhere. Another could make it easy to deploy distributed apps, but only to x86 Linux. wasmCloud has both value props :)